.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - mvn test --batch-mode
+  - mvn checkstyle:check --batch-mode --fail-never
```

Comment:
Sorry, not grokking the `--fail-never`. Especially since [the example build](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#failOnViolation) failed for style. :smile:

I *do* think style violation should (late) fail the build.